### ISSUE TITLE
Urgent - Duplicate Property ID Found in DataWarehouse 'Properties Table'

### ISSUE THREAD

**Issue by kmacgregorPR** [reapit/foundations#9724]
Power BI refreshes failing due to a duplicate Property ID being found in the Properties Table from DataWarehouse Connection. 

We have identified the property and ID, but require this to be removed as soon as possible in order for our connections to Power BI. 

ID: 59bee6a5c5454140752381ca3b98c558 - Flat 6, 30 Grosvenor Road, Jesmond, Newcastle upon Tyne, Tyne and Wear, NE2 2RP - Property Code: JES230006

Attached screenshot of duplicate within DataWarehouse 'Properties' Table. 
![Duplicate Property in DataWarehouse](https://github.com/reapit/foundations/assets/128388971/55ab8f74-6e18-4800-be79-ca5e01f32b2d)

**Comment by plittlewood-rpt** [comment]
Hi @kmacgregorPR this was caused by duplicate entries in the database for the property manager for this record. It's likely left over from when the data conversion was done for the customer. I have cleaned these up and the problem should rectify itself when the analytics schema data next refreshes at 9am. In the meantime I'll add some safeguards in place to ensure that duplicate entries don't cause this problem in the future
